CHANGELOG.md

```diff
@@ -40,6 +40,7 @@ BUG FIXES:
     the client [GH-1641]
   * discovery/jobspec: Validate service name after interpolation [GH-1852]
   * driver/docker: Fix `local/` directory mount into container [GH-1830]
+  * jobspec: Tasks without a resource block no longer fail to validate [GH-1864]
```

Comment:
Yes! Honestly this change is probably only useful for testing as the defaults really aren't intended to fit real world workflows.